projects/schema-form/src/lib/model/formproperty.ts

```diff
@@ -220,9 +220,6 @@ export abstract class FormProperty {
     this._visible = visible;
     this._visibilityChanges.next(visible);
     this.updateValueAndValidity();
-    if (this.parent) {
-      this.parent.updateValueAndValidity(false, true);
```

Comment:
As I see this is already done inside the line above `this.updateValueAndValidity();` (222)